src/devhub/entity/post/List.jsx

```diff
@@ -153,6 +153,8 @@ function defaultRenderItem(postId, additionalProps) {
           onDraftStateChange,
           ...additionalProps,
           referral: postId,
+          hideTestPost: props.tag !== "devhub-test",
```

Comment:
See the [PR here](https://github.com/near/neardevhub-widgets/pull/400/files#diff-0232ce282bd54ecead7451ede7726988c82b7fbdfc708465c44cde3788ae65d9) for how we hide Blog posts from the feed.

This can be done the same way

Reply:
Ah thanks, but then how could they access it?

Reply:
I don't think we need them to be searchable from the main feed -- they can be ignored just like blogs are.

Posts with devhub-test will always be accessible from the [devhub-test community](https://near.social/devhub.near/widget/app?page=community&handle=devhub-test)

Reply:
Got it, thanks for help. Let me fix this asap